### PR TITLE
Added option to load settings from config.ini

### PR DESCRIPTION
**Pull request by JZ6**
This pull request includes a

- [ ] Bug fix
- [x ] New feature
- [ ] Translation

The following changes were made

- Added parser load from config.ini.
-moved credentials to /config
-

If this is related to an existing ticket, include a link to it as well.

### CODE REVIEW

File: config/config.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Provide Pokemon Go Credentials here. This section must be filled out!
+
+[Athentication]
```
Comment (Ashex1) [inline config/config.ini]: Should be `Authentication`, would also be good to change `Auth_Service` to just `Service` as Auth is a bit redundant.

Comment (JZ6) [inline config/config.ini]: on it

**Comment by npeep** [comment]
Is there a way to tell the runserver.py to use this config?  I get the following after filling out the Config.ini and running the runserver.py with no arguments:

runserver.py: error: argument -u/--username is required

**Comment by npeep** [comment]
I see the -se option that was added now, thanks!!

**Comment by hhu94** [comment]
Why was this merged? It doesn't work properly. Some of the arguments are still required even when using the -se argument.

**Pull request merged**